index.html

```diff
@@ -7,16 +7,37 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome Books</title>
   <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
 </head>
 
 <body>
-  <h1 class="catch">Awesome Books</h1>
+<div>
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">Programming</a>
+  <a href="#contact">Space Sciences</a>
+  <a href="#about">About</a>
+</div>
+</div>
   <div class="container">
-    <h1 class="display"></h1>
-    <div class="books">
+    <div class="display">
+      <h1>All Awesome Books</h1>
     </div>
+    <table class="table table-striped books">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Title</th>
+          <th scope="col">Author</th>
+          <th scope="col"></th>
+        </tr>
+      </thead>
+      <tbody>
+      </tbody>
+    </table>
     <br>
     <form id="book-form">

```

Comment:
- [ ] Add a title "Add a new book" as the original layout, on top of the form, if you feel super savage, add an "hr" on top of that title.